### ISSUE TITLE
M3 Zine ideas

### ISSUE THREAD

**Issue by madjin** [open-metaverse-gathering/proposals#19]
## Name TBD

Pixel helped me come up with an overall structure for a M3 zine, check it out below!

---

It's great that you're inspired to start your own zine! Based on your interests, here's a suggested high-level table of contents for your tech art/metaverse hacker/crypto culture zine:

1. Cover Page
2. Intro/Editor's Note (1 page)
3. Table of Contents (1 page)
4. Metaverse: News, Developments, and Trends (4 pages)
    - Latest events in the metaverse
    - Profiles of emerging tech artists
    - Virtual reality and augmented reality updates
5. Hacker's Corner (4 pages)
    - In-depth tutorials and how-tos
    - Digital security best practices
6. Crypto Culture (4 pages)
    - Crypto news and analysis (Bankless plug)
    - Cryptoart plugs (M3 and friends NFTs)
    - NFT marketplaces and trends
7. Full Spread Art (4 pages)
    - Digital and traditional artwork inspired by tech, the metaverse, and hacker culture (M3 artists)
8. Retro Nostalgia (4 pages)
    - Retro gaming and vintage tech
    - Throwback to hacker zines
    - 90s / web 1.0 inspiration
    - Internet Archiving
9. DIY Tech Projects (4 pages)
    - Step-by-step guides for creating tech art
    - Arduino, Raspberry Pi, and other hardware projects
10. Community Showcase (4 pages)
    - Profiles of zine readers and their creations
    - Events, meetups, and conferences
11. Advertisements (2 pages)
    - Relevant ads for tech products, events, or services
12. Back Cover

This suggested layout includes about 32 pages, with 4 full spread art pages. You can adjust the number of pages for each section and add or remove sections based on your preferences and the content you have available. Remember, this is just a starting point, and you can customize it to make your zine unique and engaging. Good luck with your project!

**Comment by gm3** [comment]
Sounds great, can use Canva to get one started if you like

**Comment by madjin** [comment]
> Sounds great, can use Canva to get one started if you like

Yes please, do share. 7OROY wants to work on the cover as well. Reposting what was on discord here:

![image](https://user-images.githubusercontent.com/32600939/233673452-6cd1f373-516b-45bb-a9f3-7c22373edb3d.png)
